packages/n4s/src/rules/isBetween/index.js

```diff
@@ -0,0 +1,11 @@
+import gte from '../greaterThanOrEquals';
+import lte from'../lessThanOrEquals';
+
+function isBetween(value, arg1, arg2) {
+    const isCorrectType = typeof value === 'number' && typeof arg1 === 'number' && typeof arg2 === 'number';
+    return isCorrectType && (arg1 <= arg2 ? gte(value, arg1) && lte(value, arg2) : gte(value, arg2) && lte (value, arg1));
+}
```

Comment:
Wouldn't this be sufficient?
```suggestion
function isBetween(value, arg1, arg2) {
  return gte(value, arg1) && lte(value, arg2);
}
```

We don't really are about the `typeof`, the value or the args could be numeric strings (`"10"`), and both lte and gte already do the parsing.

And regarding the second part, I understand it is intended to allow flipping the numbers, but just for brevity, I think we could safely assume `isBetween(value, min, max)`. I believe 90% of the consumer will assume that immediately, and the other 10% could rely on the docs to tell them how to construct it.

Reply:
I agree that the `typeof` is redundant,  I did not assume a `min, max` I can remove the second use case. will do!